coq.opam.docker

```diff
@@ -1,42 +0,0 @@
-synopsis: "The Coq Proof Assistant"
```

Comment:
Please coordinate with the maintainer of Docker images @erikmd before merging that.